README.md

```diff
@@ -151,6 +196,20 @@ And to run all checks and tests manually, use the following command:
 poetry run pre-commit run --all-files
 ```
 
+### Linting
+
+Before committing, it is recommended to run the linter to ensure code quality:
+
+```bash
+poetry run ruff check .
+```
+
+To automatically fix linting issues:
+
+```bash
+poetry run ruff check . --fix
+```
+
```

Comment:
This is also part of the pre-commit check, please leave this out of this PR for now.

./Klaas

```suggestion
```